### PR TITLE
Update without_class.rst

### PR DESCRIPTION
**Pull request by BenjaminPorquet**
A right bracket is missing

### CODE REVIEW

File: form/without_class.rst [hunk 1]
```diff
@@ -105,4 +105,4 @@ but here's a short example:
 
 .. code-block:: php
 
-    new NotBlank(array('groups' => array('create', 'update'))
+    new NotBlank(array('groups' => array('create', 'update')))
```
Comment (xabbuh) [inline form/without_class.rst]: Shouldn't the code block be part of the tip? I suggest to indent everything starting with line 106 by additional four spaces.

**Comment by Benjamin-Porquet** [comment]
@xabbuh : like that ?

File: form/without_class.rst [hunk 2]
```diff
@@ -103,6 +103,4 @@ but here's a short example:
     ``Default`` group when creating the form, or set the correct group on
     the constraint you are adding.
 
-.. code-block:: php
-
-    new NotBlank(array('groups' => array('create', 'update')))
+        new NotBlank(array('groups' => array('create', 'update')))
```
Comment (HeahDude) [inline form/without_class.rst]: You can remove the `.. code-block::` notation but you then need to use the default php one by adding a double colon `::` at the end of the line 104.

Comment (HeahDude) [inline form/without_class.rst]: (for another language the notation should have been kept and indented as well)

**Review by HeahDude — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
Thank you @Benjamin-Porquet.

**Pull request closed without merging**